templates/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>STEP-Capstone-2020</title>
+		<link rel="stylesheet" href="{{ url_for('static', filename='style.css') }}">
+	</head>
+	<body>
+		<!--Navigation Bar-->
+		<nav>
+			<div class="logo">
+				<h4>Google STEP Capstone 2020</h4> 
+			</div>
+			<ul> 
+				<li><a href="research.html">Research</a></li> 
+			</ul>
+		</nav>
+		
+		<!--Web Title-->
+		<div class="center-title">
+			<h1 class="title-txt">Data Analysis & ML Project</h1>
+			<h6 class="tools-txt">Access Tools Below</h6>
+		</div>
+
+		<!--Tool Cards--> 
+		<div class="container">
+			<div class="box">
+				<div class="box-row">
+					<div class="box-cell card">
+						<div class="card-body">
+							<h5 class="card-title">User Hierachy Tree</h5>
+							<p class="card-text">This card will display a small description of this tool and also be the link to the tool itself.</p>
```

Comment:
Is there a way to keep these lines shorter to improve readability? Its not a must but could be something to consider. 

Also it looks like you are using tabs. You can switch VIM to use 4 spaces instead of 8 or tabs. This will also help keep the files thinner and more readable.

Reply:
Will do, I will make sure to edit these files for the next PR. I will also change VIM to be 4 spaces than 8.